assessments/projects/custodian/joint-review.md

```diff
@@ -0,0 +1,923 @@
+<!-- cSpell:ignore attck addgetnetgrent JPMC -->
+# Cloud Custodian Joint-review
+
+This joint-review relied heavily on the
+[self-assessment](https://docs.google.com/document/d/1s88ifDtFJzGX1O_ve6HM6_vbTkpsWDKWRDuxt9RHdcs)
+provided by the Cloud Custodian (c7n) project. The Security TAG volunteers
+collaborated with the project volunteers to review the self-assessment, and
+discussed the operational scenarios pertinent to a typical c7n user.
+
+The general opinion of the volunteer review team is that the c7n project
+clearly demonstrates a commitment to security, and a willingness to improve
+security for the benefit of the user community.  That said, the project
+volunteers could use additional security volunteers to assist in ongoing
+reviews, security control enhancements, and when needed to debug and fix
+security flaws.  As a utility tool, the review team feels the primary threat to
+the project is compromise of the code itself, and packaging and distribution of
+the tool or its dependencies - rather than an attack on individual c7n
```

Comment:
I'm curious what led to this conclusion since we haven't done a code review or tried out a sample deployment.